core/app/models/spree/ability.rb

```diff
@@ -42,19 +42,15 @@ def initialize(user)
         can :update, Order do |order, token|
           !order.completed? && (order.user == user || order.token && token == order.token)
         end
-        can :manage, Spree::Address do |address|
-          address.user == user
-        end
-        can :create, Spree::Address do |_address|
-          user.id.present?
-        end
+        can :manage, Address, user_id: user.id
```

Comment:
CodeClimate throws `Method "initialize" has 34 lines of code (exceeds 25 allowed). Consider refactoring.` so I would change `can :read` method to inline: `can :read, [State, Store, Taxon...]`

Reply:
@KacperMekarski yup, but we don't want to modify that ability class too much to not lose the git history behind